### PR TITLE
Fix canonical

### PR DESCRIPTION
**Pull request by mhessdev**
had to add Head to the index js to override the canonical being set by config. This seemed the only way to do this.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,17 +1,8 @@
 {
-    "printWidth": 100,
-    "jsxBracketSameLine": false,
-    "singleQuote": true,
-    "tabWidth": 2,
-    "trailingComma": "es5",
-    "semi": false,
-    "plugins": ["prettier-plugin-prisma"],
```
Comment (petradonka) [inline .prettierrc:8]: plugins is removed also, intentional?

**Comment by petradonka** [comment]
Changes seem good to me, but I'd like someone else to double-check and approve

File: .prettierrc (hunk 1 above)
Comment (mhessdev) [reply .prettierrc:8]: Yes, pretty sure we don't need it. Correct @jharrell ?

Comment (jharrell) [reply .prettierrc:8]: correct

**Review by jharrell — APPROVED** [review]

**Pull request merged**